libs/template/application/package.json

```diff
@@ -18,7 +18,9 @@
     "@webcomponents/template-shadowroot": "^0.1.0",
     "abort-controller": "^3.0.0",
     "express": "^4.17.1",
-    "url-join": "^5.0.0"
+    "url-join": "^5.0.0",
+    "workbox-core": "^6.5.4",
+    "workbox-precaching": "^6.5.4"
```

Comment:
I'd love to avoid this. Workbox is only used for apps that need sw, but storefront currently does not need this, and customers might not need this going forward. Can we not move/keep this at a lower level package instead?

Reply:
I'd leave it as it is for now. `offline` package doesn't seem right for it, new package is also not so good option, so in case we want to reduce our boilerplate we need this code here.